frontend/test/ui/pages/desktop/base.py

```diff
@@ -0,0 +1,17 @@
+from pypom import Page
+
+from selenium.webdriver.common.by import By
+
+
+class Base(Page):
+
+    _copter_locator = (By.CLASS_NAME, 'copter')
+
+    def __init__(self, selenium, base_url, locale='en-US', **kwargs):
```

Comment:
Are the pages localised, or is `locale` an artifact of copy/paste from another project? I don't see it used anywhere in this patch.